light-chat/index.js

```diff
@@ -0,0 +1,166 @@
+import * as utils from "https://unpkg.com/@waku/byte-utils@0.0.2/bundle/index.js";
+import * as wakuCreate from "https://unpkg.com/@waku/create@0.0.4/bundle/index.js";
+import { waitForRemotePeer } from "https://unpkg.com/@waku/core@0.0.6/bundle/lib/wait_for_remote_peer.js";
+import * as wakuMessage from "https://unpkg.com/@waku/core@0.0.6/bundle/lib/waku_message/version_0.js";
+
+const MULTI_ADDR = "/dns4/node-01.ac-cn-hongkong-c.wakuv2.test.statusim.net/tcp/443/wss/p2p/16Uiu2HAkvWiyFsgRhuJEb9JfjYxEkoHLgnUQmr1N5mKWnYjxYRVm";
+const CONTENT_TOPIC = "/toy-chat/2/huilong/proto";
+const PROTOCOLS = ["filter", "lightpush"];
+
+runApp().catch((err) => {
+    console.error(err);
+});
+
+async function runApp() {
+    const ui = initUI();
+
+    ui.setStatus("connecting...");
+
+    const { info, sendMessage, unsubscribeFromMessages } = await initWakuContext({
+        protocols: PROTOCOLS,
+        multiAddr: MULTI_ADDR,
+        contentTopic: CONTENT_TOPIC,
+        onMessageReceived: ui.renderMessage,
+    });
+
+    ui.setStatus("connected");
+
+    ui.setLocalPeer(info.localPeerId);
+    ui.setRemotePeer(info.remotePeerId);
+    ui.setRemoteMultiAddr(info.multiAddr);
+    ui.setContentTopic(info.contentTopic);
+
+    ui.onSendMessage(sendMessage);
+
+    ui.onExit(async () => {
+        ui.setStatus("disconnecting...");
+        await unsubscribeFromMessages();
+        ui.setStatus("disconnected");
+        ui.resetMessages();
+    });
+}
+
+async function initWakuContext({
+    multiAddr,
+    protocols,
+    contentTopic,
+    onMessageReceived,
+}) {
+    const Decoder = new wakuMessage.DecoderV0(contentTopic);
+    const Encoder = new wakuMessage.EncoderV0(contentTopic);
+
+    const ChatMessage = new protobuf.Type("ChatMessage")
+        .add(new protobuf.Field("timestamp", 1, "uint64"))
+        .add(new protobuf.Field("nick", 2, "string"))
+        .add(new protobuf.Field("text", 3, "bytes"));
+
+    const node = await wakuCreate.createLightNode();
```

Comment:
```suggestion
    const node = await wakuCreate.createLightNode( { defaultBootstrap: true } );
```

This is the recommended way to get started :)